client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -0,0 +1,390 @@
+					modalTitle={ translate( 'You’re about to update your production site' ) }
+					extraModalContent={
+						<div>
+							<p>
+								{ translate(
+									'Synchronizing your production site will overwrite the following items with their equivalents from the staging site:'
+								) }
+							</p>
+							<ConfirmationModalList>
+								{ selectedItems.map( ( item ) => {
+									return <li key={ item }>{ item }</li>;
+								} ) }
+							</ConfirmationModalList>
+							<ConfirmationModalInputTitle>
+								{ translate( "Enter your site's name {{span}}%(siteSlug)s{{/span}} to confirm.", {
+									args: {
+										siteSlug: siteSlug as string,
+									},
+									components: {
+										span: <ConfirmationModalInputSiteSlug />,
+									},
+								} ) }
+							</ConfirmationModalInputTitle>
+							<ConfirmationModalInput
+								value={ typedSiteName }
+								onChange={ ( event: ChangeEvent ) => setTypedSiteName( event.target.value ) }
+							/>
+						</div>
+					}
+					confirmLabel={ translate( 'Synchronize' ) }
+					cancelLabel={ translate( 'Cancel' ) }
+				>
+					<span>{ translate( 'Synchronize' ) }</span>
+				</ConfirmationModal>
+			</ConfirmationModalContainer>
+		</>
+	);
+};
+
+const ProductionToStagingSync = ( {
+	disabled,
+	isSyncButtonDisabled,
+	onConfirm,
+}: {
+	disabled: boolean;
+	isSyncButtonDisabled: boolean;
+	onConfirm: () => void;
+} ) => {
+	return (
+		<ConfirmationModalContainer>
+			<ConfirmationModal
+				disabled={ disabled || isSyncButtonDisabled }
+				isPrimary={ true }
+				onConfirm={ onConfirm }
+				modalTitle={ translate( 'You are about to update your staging site' ) }
+				modalMessage={ translate(
+					'Synchronizing your staging site will replace the contents of the staging site with those of your production site.'
+				) }
+				confirmLabel={ translate( 'Synchronize' ) }
+				cancelLabel={ translate( 'Cancel' ) }
+			>
+				<span>{ translate( 'Synchronize' ) }</span>
+			</ConfirmationModal>
+		</ConfirmationModalContainer>
+	);
+};
+
+const SyncCardContainer = ( { children } ) => {
+	const translate = useTranslate();
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	return (
+		<StagingSyncCardBody>
+			<SyncContainerTitle>{ translate( 'Data and File synchronization' ) }</SyncContainerTitle>
+			<SyncContainerContent>
+				{ translate(
+					'Keep your database and files synchronized between the production and staging environments.'
+				) }
+			</SyncContainerContent>
+			<SyncContainerTitle>{ translate( 'Choose synchronization direction' ) }</SyncContainerTitle>
+			{ children }
+			<StagingSyncCardFooter>
+				{ translate(
+					"We'll automatically back up your site before synchronization starts. Need to restore a backup? Head to the {{link}}Activity Log.{{/link}}",
+					{
+						components: {
+							link: (
+								<a
+									href={ `/activity-log/${ siteSlug }` }
+									target="_blank"
+									rel="noopener noreferrer"
```

Comment:
Hm, I think we should remove `target` and `rel` if it's an internal link.

Reply:
Daniel, I thought about that as well. I find it easier ( but this could be just me ) to open the page to a new tab, restore a backup, and move back to the other tab to initiate syncing. ( instead of navigating through the menu ). 
However, I investigated it a bit, and you are right. For internal links, we don't open pages in new tab. 
Thanks for your comment. This is fixed.